cves/CVE-2011-1772.yml

```diff
@@ -103,9 +103,9 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: The original code was unit tested, but was not properly tested with input vectors to check for vulnerabilities.
```

Comment:
Did the fix involve adding more unit tests or improving existing unit tests?

Reply:
No tests were changed regarding this fix, and that's been updated in the most recent commit.